examples/mne_inverse_envelope_correlation.py

```diff
@@ -82,6 +82,9 @@
     stcs, labels, inv['src'], return_generator=True)
 corr = envelope_correlation(label_ts, verbose=True)
 
+# average over epochs
+corr = corr.combine(combine='mean')
```

Comment:
The default is `'mean'` so I think this can just be
```suggestion
corr = corr.combine()
```